### PR TITLE
Make database take an argument of db name when making a collection

### PR DESCRIPTION
**Pull request by zhangxin511**
Add the ability to be able to take a db name when making a collection

### CODE REVIEW

**Comment by yangg-msft** [comment]
Probably we can add some descriptions about why we'd like to add the extra parameter?

**Review by yangg-msft — APPROVED** [review]
:shipit:

**Review by pradeepvairamani — APPROVED** [review]

File: server/routerlicious/packages/services-core/src/database.ts [hunk 1]
```diff
@@ -263,7 +263,7 @@ export interface IDb {
 
 	on(event: IDbEvents, listener: (...args: any[]) => void);
 
-	collection<T>(name: string): ICollection<T>;
+	collection<T>(name: string, dbName?: string): ICollection<T>;
```
Comment (hedasilv) [inline server/routerlicious/packages/services-core/src/database.ts:266]: Does the associated implementation of this interface need to change to match this interface update?

Comment (hedasilv) [reply server/routerlicious/packages/services-core/src/database.ts:266]: Oh, I guess it already takes that parameter into account:
https://github.com/microsoft/FluidFramework/blob/500e404c31c9ce26da4696c9a2f45fb624be2ad3/server/routerlicious/packages/services/src/mongodb.ts#L514

**Pull request merged**